### PR TITLE
github: add pull request template

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+**Tasks**
+- [ ] Documentation (docs and inline)
+- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
+- [ ] Changelog
```
Comment (philli-m) [inline .github/pull_request_template.md:4]: should we also add naming convention?

**Review by philli-m — APPROVED** [review]
if my question doesn't make sense then please merge

File: .github/pull_request_template.md (hunk 1 above)
Comment (goapunk) [reply .github/pull_request_template.md:4]: @philli-m sorry, not sure what you mean - naming convention for what?

Comment (philli-m) [reply .github/pull_request_template.md:4]: @goapunk naming convention of the pull request, with story no or ref and short description, helps finding them again

**Review by philli-m — APPROVED** [review]
Looks good!

**Pull request merged**